include/GL/glui.h

```diff
@@ -442,7 +444,7 @@ class GLUIAPI GLUI_StdBitmaps
 {
 public:
     GLUI_StdBitmaps(); 
-    ~GLUI_StdBitmaps();
+    ~GLUI_StdBitmaps() = default;
 
```

Comment:
I'm leaning towards a GLUI 3.0 being C++11 onwards.

`public ~GLUI_StdBitmaps() = default` at least clarifies that there is nothing special or constrained about lifetime management.